public/scripts/templates/themeDelete.html

```diff
@@ -1,3 +1,4 @@
 <div>
-    Are you sure you want to delete the theme "{{themeName}}"?
+    <h3 data-i18n="Are you sure you want to delete the theme?">Are you sure you want to delete the theme?</h3>
```

Comment:
I adjusted the layout to avoid translation issues with variables like `{{name}}`, especially in Chinese. Separating them improves readability and compatibility with data-i18n.

If you prefer, I can keep the original layout instead!